CODEOWNERS

```diff
@@ -0,0 +1,11 @@
+* @bpfd-dev/bpfd-maintainers
+
+./bpfd @dave-tucker
+./bpfd/src/oci_utils @astoycos
+./bpfd/src/multiprog/tc.rs @anfredette
+./bpfd/src/certs.rs @Billy99
+./bpfd-api @dave-tucker
+./bpfctl @dave-tucker
+./bpfd-operator @astoycos
+./examples/go-tc-counter @anfredette
+./examples/go-xdp-counter @Billy99
```

Comment:
If they don't it will fall back to `bpfd-maintainers`, which is the 4 of us.
Figured we'd just add overrides if one person has more contributions in a given area.